DESCRIPTION

```diff
@@ -11,7 +11,8 @@ Depends:
 Imports:
     AWS.tools,
     stringr,
-    digest
+    digest,
```

Comment:
Can you list the packages in alphabetical order? This is the typical convention but wasn't implemented as of the creation of this package.